siteapp/views.py

```diff
@@ -1514,37 +1516,41 @@ def preload_profiles(users):
 def organization_settings_save(request):
     if request.method != "POST":
         return HttpResponseForbidden()
-    if request.user not in request.organization.get_organization_project().get_admins():
-        return HttpResponseForbidden()
+
+    # In 0.9.0 we only have 1 organization, so let's get that
+    # TODO better setting of organization
+    organization = Organization.objects.get(id=1)
```

Comment:
I would do something like `Organization.objects.get()`. If there is one in the database, it will return it. If there is zero or more than one, it will raise an exception. This way if the assumption is wrong, there's an error, rather than possibly editing the wrong Organization instance.

Reply:
Well, there is a chance in the future we could go back to having more than 1 organization. During install, the first organization created is the main organization.

Reply:
All the more reason to have it not silently take Organization 1.